lib/dnsimple/struct/email_forward.rb

```diff
@@ -11,15 +11,23 @@ class EmailForward < Base
       attr_accessor :domain_id
 
       # @return [String] The "local part" of the originating email address. Anything to the left of the @ symbol.
+      # DEPRECATED: use `alias_email` instead
```

Comment:
We probably should `@deprecated` https://rubydoc.info/gems/yard/file/docs/Tags.md#deprecated

Reply:
Do we actually use YARD in this codebase? I couldn't find it in the gemfile...

Reply:
🤔 we may not be linting, or generating docs from it, but I'd say that's the format we are using for all ruby repos... The `@param` and `@return` are all [YARD](https://yardoc.org/)...